test-project/Assets/EditmodeTests/GameObjectRepresentation/MonoBehaviourActivationManagerTests.cs

```diff
@@ -0,0 +1,347 @@
+using Generated.Improbable;
+using Generated.Improbable.Gdk.Tests.ComponentsWithNoFields;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.Commands;
+using Improbable.Worker.Core;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.GameObjectRepresentation.EditModeTests.MonoBehaviourActivationManagerTests
+{
+    public abstract class ActivationManagerTestBase<TBehaviour> where TBehaviour : Component
```

Comment:
Why not `TBehaviour : MonoBehaviour`?

Reply:
it was happy with `Component`, but will make it more specific